tools/travis/runSystemTests.sh

```diff
@@ -19,18 +19,18 @@
 
 set -e
 
-SCRIPTDIR=$(cd $(dirname "$0") && pwd)
+SCRIPTDIR=$(cd "$(dirname "$0")" && pwd)
 ROOTDIR="$SCRIPTDIR/../.."
 
 cd $ROOTDIR/tools/travis
 
 export ORG_GRADLE_PROJECT_testSetName="REQUIRE_SYSTEM"
 export GRADLE_COVERAGE=true
 
-./setupPrereq.sh /ansible/files/runtimes-nodeonly.json
+./setupPrereq.sh /ansible/files/runtimes-nodeonly.json 240
```

Comment:
The current default is 60 - won't setting this to 240 cause any tests that might rely on 60 to fail?
When does travis run on 'faster' machines? Is there an example failure at https://travis-ci.org/apache/incubator-openwhisk/?